### PR TITLE
Use ZEN remote control in ACE interaction menu action

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- title, replace ACE using BI's remote control with own

### CODE REVIEW

File: optionals/compat_ace/functions/fnc_canRemoteControl.sqf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * Example:
+ * [_curator] call zen_compat_ace_fnc_remoteControl
+ *
+ * Public: No
+ */
+
+params ["_curator"];
+
+SELECTED_OBJECTS findIf {_x call EFUNC(remote_control,canControl)} != -1
+&& {[_curator, {"ace_interaction" in curatorAddons _this}, missionNamespace, QACEGVAR(interaction,zeusCheck), 1e9, "ace_interactMenuClosed"] call ACEFUNC(common,cachedCall)}
```
Comment (CreepPork) [inline optionals/compat_ace/functions/fnc_canRemoteControl.sqf:21]: This whole statement is very hard to understand. Could it be separated into different statements with comments or something to make it less confusing (there's code, namespaces, 1e9 and so on).

Comment (mharis001) [reply optionals/compat_ace/functions/fnc_canRemoteControl.sqf:21]: I can see how on first read-though it looks weird but that is just how the arguments for `cachedCall` are. Can't really be changed.

Comment (CreepPork) [reply optionals/compat_ace/functions/fnc_canRemoteControl.sqf:21]: The arguments could be extracted into separate variables which would try to achieve named parameters.

So you wouldn't need to look up what would be the arguments of that function but just look how the parameters are named.

Comment (mharis001) [reply optionals/compat_ace/functions/fnc_canRemoteControl.sqf:21]: I think that is unnecessary for this situation (or for which argument it is helpful, you just have to know how the function works).

**Pull request merged**